pkg/scheduler/algorithm/predicates/metadata.go

```diff
@@ -421,7 +421,7 @@ func getPodsMatchingAffinity(pod *v1.Pod, nodeInfoMap map[string]*schedulercache
 			appendResult(node.Name, nodeTopologyPairsAffinityPodsMaps, nodeTopologyPairsAntiAffinityPodsMaps)
 		}
 	}
-	workqueue.Parallelize(16, len(allNodeNames), processNode)
+	workqueue.ParallelizeUntil(nil, 16, len(allNodeNames), processNode)
```

Comment:
is a nil context an anti-pattern? Never seen that. There is `context.Background()` and `context.TODO()`.

Reply:
these cases in the scheduler all look like one of those two.

Reply:
yeah, you are right. The default will be context.TODO(), but not nil. But ParallelizeUntil will handle it when the context is nil.